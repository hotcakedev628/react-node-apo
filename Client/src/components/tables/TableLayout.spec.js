import renderer from 'react-test-renderer';
import {TableLayout} from './TableLayout';
import React from 'react'



describe('TableLayout', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<TableLayout 
        TableHeaders={()=>(<li>HH</li>)}
        TableBodyContent={()=>(<li>C</li>)}
        />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
