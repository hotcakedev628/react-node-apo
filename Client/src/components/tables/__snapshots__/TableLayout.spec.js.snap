// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableLayout should match snapshot 1`] = `
<div
  className="mt-4"
>
  <table
    className="table table-hover"
  >
    <thead>
      <tr>
        <li>
          HH
        </li>
      </tr>
    </thead>
    <tbody>
      <li>
        C
      </li>
    </tbody>
  </table>
</div>
`;
